scripts/build.sh

```diff
@@ -46,3 +46,7 @@ export GOOS=linux
 export GOARCH=amd64
 mkdir -p "${build_dir}/${GOOS}/x86_64"
 "${go_build[@]}" -o "${build_dir}/${GOOS}/x86_64/${plugin_name}" . || exit 1
+export GOOS=darwin
```

Comment:
Darwin compatibility should be done by editing [.sync.yml](https://github.com/intelsdi-x/snap-plugin-collector-apache/blob/master/.sync.yml) and running pluginsync.